src/commands/Utility/quote.ts

```diff
@@ -40,6 +40,12 @@ const command: Command = {
 			name: "url",
 			description: "The url of the message this quote came from",
 			required: false
+		},
+		{
+			type: "STRING",
+			name: "pictureUrl",
+			description: "The url of the picture to be included with this quote",
```

Comment:
Current name is invalid. See [this](https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-naming)
```suggestion
			name: "image",
			description: "The url of the image to be included with this quote",
```

Reply:
Fair enough